deploy/template.yaml

```diff
@@ -83,25 +83,25 @@ parameters:
       {
         "openshift_version": "4.13",
         "cpu_architecture": "x86_64",
-        "url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/pre-release/latest-4.13/rhcos-4.13.0-ec.3-x86_64-live.x86_64.iso",
+        "url": "https://mirror.openshift.com/pub/openshift-v4/x86_64/dependencies/rhcos/pre-release/4.13.0-rc.2/rhcos-4.13.0-rc.2-x86_64-live.x86_64.iso",
```

Comment:
Do we need to update anything in the assisted service?

Reply:
Nope, they were updated in https://github.com/openshift/assisted-service/pull/5115